.github/workflows/pre-commit.yml

```diff
@@ -0,0 +1,30 @@
+name: "Pre commit"
```

Comment:
I used a Github workflow for which I had a snippet. This repo uses circleci with which I have no experience we could port it to there.